src/components/Contributors/Contributors.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Grid from '@material-ui/core/Grid';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+import { Team } from 'types';
+
+interface Props {
+    team: Team;
+    maxDisplayCount: number;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        large: {
+            width: theme.spacing(7),
+            height: theme.spacing(7),
+        },
+    })
+);
+export default function Contributors({ team, maxDisplayCount }: Props) {
+    const classes = useStyles();
+    return (
+        <Grid container spacing={5}>
+            <Grid
+                container
+                item
+                justify='center'
+                alignItems='center'
+                spacing={3}
+            >
+                {team.members.slice(0, maxDisplayCount).map((member, index) => (
+                    <Grid item xs={6} sm={2} key={index}>
+                        <Grid
+                            container
+                            direction='column'
+                            alignItems='center'
+                            justify='center'
+                        >
+                            <Grid item>
+                                <Avatar
+                                    className={classes.large}
+                                    key={index}
+                                    alt={member.fullName}
+                                    src={member.picturePath}
+                                />
+                            </Grid>
+                            <Grid item>
+                                <Typography
+                                    align='center'
+                                    color='textSecondary'
+                                    variant='caption'
+                                >
+                                    {member.fullName}
+                                </Typography>
+                            </Grid>
+                        </Grid>
+                    </Grid>
+                ))}
+            </Grid>
+            <Grid container item justify='center'>
+                {/* TODO: When clicked redirect to a page that displays the entire team using the Team component  */}
+                <Link href='www.todo.add.href'>
+                    {`+ ${team.members.length - maxDisplayCount} Contributors`}
```

Comment:
One more thing I noticed now that we're using `maxDisplayCount`.  If, for some reason, the dev makes it such that `maxDisplayCount > team.members.length` (team.members.length is an API call, so we technically don't always know it) then this will display a negative number.  So, `Math.max(team.members.length - maxDisplayCount, 0)` should fix this.